app/models/workflow.rb

```diff
@@ -80,7 +80,7 @@ def adjust_sequences
 
```

Comment:
How about we `return` from here directly for this case? I feel it's more clear than current fix

Reply:
it is a temporary fix. The whole logic is going away soon.